File: src/main/java/pl/training/workshop/m7/s13_godclass/step4/Formats.java
package pl.training.workshop.m7.s13_godclass.step4;

import java.util.Locale;

/** Krok 2: format kwot ("114.00") potrzebny i CinemaManager, i NotificationService - jeden właściciel. */
final class Formats {
    private Formats() {
    }

    static String amount(double value) {
        return String.format(Locale.ROOT, "%.2f", value);
    }
}

File: csharp/src/Training.Workshop/M7/S13GodClass/Step4/Formats.cs
using System.Globalization;

namespace Training.Workshop.M7.S13GodClass.Step4;

/// <summary>Krok 2: format kwot ("114.00") potrzebny i CinemaManager, i NotificationService - jeden właściciel.</summary>
internal static class Formats
{
    internal static string Amount(double value)
    {
        return value.ToString("F2", CultureInfo.InvariantCulture);
    }
}
